server/server/src/metrics.rs

```diff
@@ -159,7 +188,7 @@ pub async fn calculate_total_document_bytes(
     let mut total_size: u64 = 0;
 
     for metadata in metadatas {
-        if metadata.file_type == FileType::Document {
+        if metadata.file_type == FileType::Document && metadata.content_version != 0 {
             let file_usage: FileUsage = con
```

Comment:
need a sleep here

Reply:
also we still need to not error out here, a file may be deleted and no longer have a `size` while this function is running, metrics shouldn't abort because of this.